Lib/cProfile.py

```diff
@@ -37,6 +47,12 @@ class Profile(_lsprof.Profiler):
     # Most of the functionality is in the base class.
     # This subclass only adds convenient and backward-compatible methods.
 
+    def __enter__(self):
+        self.enable()
```

Comment:
@ncoghlan also, should these __enter__ add the print option? or add like __init__ with print option.